frontend/components/droplets/lessons/lesson-renderer.tsx

```
@@ -334,10 +334,13 @@ function convertSingleBlock(blockAny: any, blockIndex: number): Block | null {
         default: "bg-sky-50 dark:bg-sky-200",
       };
 
-      const calloutContent = (blockAny.content ?? []) as any[];
+      const calloutContent = (blockAny.content ?? []) as Array<{
+        type?: string;
+        text?: string;
+      }>;
       const calloutText =
         calloutContent
-          .map((item: any) => (item.type === "text" ? item.text ?? "" : ""))
+          .map((item: any) => (item.type === "text" ? (item.text ?? "") : ""))
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)